plugins/basic/js/shotgun/constants.js

```diff
@@ -36,14 +36,14 @@ sg_constants.product_info = {
     // Photoshop
     PHSP: {
         display_name: "Photoshop",
-        tk_engine_name: "tk-photoshop",
+        tk_engine_name: "tk-photoshopcc",
```

Comment:
Thanks. It's worth keeping it here. I think this was likely looking forward to the eventuality of having multiple engines for different CC applications.